src/components/mapUrl.jsx

```diff
@@ -1,16 +1,29 @@
 import React from 'react';
+import Paper from 'material-ui/Paper'
+import muiThemeable from 'material-ui/styles/muiThemeable';
 
-export default class MapUrl extends React.Component {
+export class MapUrl extends React.Component {
   render() {
+    const background = this.props.muiTheme ? this.props.muiTheme.palette.primary1Color : '#fff';
```

Comment:
it does make test easier and if you want to use the component without the material-ui then maybe :)